apps/studio/hooks/analytics/useDbQuery.tsx

```diff
@@ -38,12 +53,12 @@ const useDbQuery = (
     isRefetching,
     refetch,
   } = useQuery(
-    ['projects', project?.ref, 'db', { ...params, sql: resolvedSql }, where, orderBy],
+    ['projects', project?.ref, 'db', { ...params, sql: resolvedSql, identifier }, where, orderBy],
     ({ signal }) => {
       return executeSql(
         {
           projectRef: project?.ref,
```

Comment:
Just want to make sure this should still be `project?.ref` and not `identifier` still

Reply:
yup so on line 45 if read replicas is enabled, it'll use the selectedDatabaseId, other default back to project.ref

in this case, each replica will have its own query performance results, hence why added the identifier param in the query key